storybook/storybook-docs/src/introduction.md

```diff
@@ -2,81 +2,103 @@
+### Reply
 
-Op deze website vind je de React Library en documentatie – gebruik de inhoudsopgave links (op brede vensters) of onder de tab ‘Sidebar’ (op smalle). Deze componenten zijn [via npm](https://www.npmjs.com/search?q=%40amsterdam%2Fdesign-system) te installeren en te gebruiken in je applicatie.
+On this website, you will find the React Library and documentation – use the table of contents on the left (on wide windows) or under the ‘Sidebar’ tab (on narrow ones).
+These components can be installed and used in your application [via npm](https://www.npmjs.com/search?q=%40amsterdam%2Fdesign-system).
 
 ### React Native
 
-Bij het ontwikkelen van de Amsterdam App zijn al veel UI-componenten gebouwd volgens de vernieuwde huisstijl. Ze staan nog [in die repository](https://github.com/Amsterdam/amsterdam-app-frontend), maar worden op termijn overgeheveld naar die van het design system zodat ze ook door andere apps gebruikt kunnen worden.
+While developing the Amsterdam App, many UI components have been built according to the updated corporate identity.
+They are still [in that repository](https://github.com/Amsterdam/amsterdam-app-frontend) but will eventually be transferred to the design system for other apps to use.
 
 ### CSS
 
-Ook zonder React zijn de componenten te gebruiken, namelijk door de juiste _classes_ toe te passen. Als je deze aanpak wil volgen horen we het graag.
+The components are usable without React by applying the correct classes.
+If you would like to follow this approach, please let us know.
 
 ### Salesforce
 
-Voor het nieuwe amsterdam.nl voorzien we een bibliotheek van Lightning Web Components in de juiste vormgeving, zodat ze gebruikt kunnen worden in Salesforce CMS.
+For the new amsterdam.nl, we provide a library of correctly designed Lightning Web Components usable in Salesforce CMS.
 
 ### Mendix
 
-Ook applicaties die in dit low-code platform worden opgetuigd helpen we op de huisstijl te laten aansluiten.
+We help applications implemented in this low-code platform match the corporate identity.
 
-## We bouwen op NL Design System
+## We build on the NL Design System
 
-Amsterdam Design System gebruikt de architectuur van [NL Design System](https://nldesignsystem.nl/).
+Amsterdam Design System uses the architecture of [NL Design System](https://nldesignsystem.nl/).
 
-Dankzij het NL Design System kan de hele overheid in Nederland samenwerken aan een begrijpelijke, gebruiksvriendelijke én toegankelijke online dienstverlening. Een dienstverlening die logisch en samenhangend is, maar niet per se uniform, want binnen de afspraken blijft er ruimte voor de eigen identiteit van overheidsorganisaties.
+Thanks to the NL Design System, the entire government in the Netherlands can work together to provide understandable, user-friendly and accessible online services.
+A service that is logical and coherent but not necessarily uniform because, within the agreements, there is room for the identity of government organizations.
```

Comment:
```suggestion
A service that is logical and coherent but not necessarily uniform, because it allows room for the identity of government organizations.
```